src/processing/spatial_operations.jl

```diff
@@ -1,38 +1,39 @@
 
 """
     simplify_exclusions!(
-    exclusions::DataFrame;
-    min_area::Real=50,
-    simplify_tol::Real=2
+        exclusions::DataFrame;
+        min_area::Float64=3E-7,
+        simplify_tol::Float64=1E-4
     )::DataFrame
 
-Simplify exclusions by removing small polygons, and simplifying geometry.
+Removing small polygons and simplify geometry based on respective tolerance values.
```

Comment:
You should preference active voice when writing documentation.

```suggestion
Remove small polygons and simplify geometry based on tolerance values.
```

The function name is misleading and doesn't align with what it does. It is doing more than "simplify exclusions", you're also removing small polygons.
In this case it's fine - it's a two-step process that you want a convenient one-liner for, I understand that.

How about `process_exclusions!` or something to that effect?

Or, `prepare_exclusion_zones!`...